src/main/flight/gps_rescue.c

```diff
@@ -194,17 +194,19 @@ static void setReturnAltitude(void)
         // Intended descent distance for rescues that start outside the minStartDistM distance
         // Set this to the user's intended descent distance, but not more than half the distance to home to ensure some fly home time
         rescueState.intent.descentDistanceM = fminf(0.5f * rescueState.sensor.distanceToHomeM, gpsRescueConfig()->descentDistanceM);
- 
+
+        const float initiaClimbCm = gpsRescueConfig()->initialClimbM * 100.0f;
```

Comment:
(IMO this is distracting - you need to look at this line to understand what initialClimbCm means. `cfgInitialClimbCm` may be slightly better if you insist on helper variable.

Also note the misspelling (initia)